File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/css/TextAlignment.java
package net.dontdrinkandroot.wicket.bootstrap.css;

import net.dontdrinkandroot.wicket.css.CssClass;


public enum TextAlignment implements CssClass
{

	LEFT("text-left"),
	CENTER("text-center"),
	RIGHT("text-right"),
	JUSTIFY("text-justify"),
	NOWRAP("text-nowrap");

	private String classString;


	private TextAlignment(String classString)
	{
		this.classString = classString;
	}

	@Override
	public String getClassString()
	{
		return this.classString;
	}
}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/css/TextAlignment.kt
package net.dontdrinkandroot.wicket.bootstrap.css

import net.dontdrinkandroot.wicket.css.CssClass

enum class TextAlignment(override val classString: String) : CssClass {
    START("text-start"),
    CENTER("text-center"),
    END("text-end"),
    JUSTIFY("text-justify"),
    NOWRAP("text-nowrap");
}